api/api.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.12
+// 	protoc        v3.20.1
```

Comment:
Is this restoring a "canonical" version for this repo, or are we happy flip flopping around here?

Reply:
Ah no, this wasn't intentional - lemme upgrade my `protoc` and regen.

Reply:
Just to capture a bit more discussion IRL: we'll merge this as-is for now, since this is the same `protoc` version used in other repos we own (e.g. Trillian), but we should probably take a separate task to go around and upgrade all the repos to a more recent version.